src/main/java/com/minecolonies/entity/ai/util/Structure.java

```diff
@@ -27,6 +27,8 @@
      */
     public static final class StructureException extends Exception
     {
+        public static final long serialVersionUID = 8632728763984762837L;
+
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Explicitly declare the visibility for "serialVersionUID". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2039)